src/main/scala/treadle/executable/SymbolTable.scala

```diff
@@ -50,7 +50,7 @@ class SymbolTable(val nameToSymbol: mutable.HashMap[String, Symbol]) {
 
   val stopToStopInfo:     mutable.HashMap[Stop, StopInfo] = new mutable.HashMap[Stop, StopInfo]
   val printToPrintInfo:   mutable.HashMap[Print, PrintInfo] = new mutable.HashMap[Print, PrintInfo]
-  val verifyToVerifyInfo: mutable.HashMap[Verification, VerifyInfo] = new mutable.HashMap[Verification, VerifyInfo]
+  val verifyInfo: mutable.HashMap[String, VerifyInfo] = new mutable.HashMap[String, VerifyInfo]
```

Comment:
Nit: does not appear to have had formatter run over it.

Reply:
How do I selectively run the formatter?
For `firrtl` I always just run `sbt scalafmtAll` but in `treadle`'s case this leads to a lot of files being modified:

```	modified:   src/main/scala/treadle/Driver.scala
	modified:   src/main/scala/treadle/Regression.scala
	modified:   src/main/scala/treadle/ScalaBlackBox.scala
	modified:   src/main/scala/treadle/TreadleOptions.scala
	modified:   src/main/scala/treadle/TreadleRepl.scala
	modified:   src/main/scala/treadle/TreadleTester.scala
	modified:   src/main/scala/treadle/VcdReplayTester.scala
	modified:   src/main/scala/treadle/coverage/Coverage.scala
	modified:   src/main/scala/treadle/coverage/package.scala
	modified:   src/main/scala/treadle/coverage/pass/AddCoverageExpressions.scala
	modified:   src/main/scala/treadle/executable/BigPrimOps.scala
	modified:   src/main/scala/treadle/executable/BlackBoxCycler.scala
	modified:   src/main/scala/treadle/executable/ClockInfo.scala
	modified:   src/main/scala/treadle/executable/ClockStepper.scala
	modified:   src/main/scala/treadle/executable/DataStore.scala
	modified:   src/main/scala/treadle/executable/IntOps.scala
	modified:   src/main/scala/treadle/executable/LongPrimOps.scala
	modified:   src/main/scala/treadle/executable/Memory.scala
	modified:   src/main/scala/treadle/executable/RenderedExpression.scala
	modified:   src/main/scala/treadle/executable/RollBackBuffer.scala
	modified:   src/main/scala/treadle/executable/Scheduler.scala
	modified:   src/main/scala/treadle/executable/SensitivityGraphBuilder.scala
	modified:   src/main/scala/treadle/executable/SymbolTable.scala
	modified:   src/main/scala/treadle/executable/Transition.scala
	modified:   src/main/scala/treadle/executable/TreadleException.scala
	modified:   src/main/scala/treadle/executable/VerifyOp.scala
	modified:   src/main/scala/treadle/executable/WaveformValues.scala
	modified:   src/main/scala/treadle/package.scala
	modified:   src/main/scala/treadle/repl/ReplConfig.scala
	modified:   src/main/scala/treadle/repl/ReplOptions.scala
	modified:   src/main/scala/treadle/repl/ReplVcdController.scala
	modified:   src/main/scala/treadle/repl/Script.scala
	modified:   src/main/scala/treadle/repl/TreadleReplStage.scala
	modified:   src/main/scala/treadle/stage/TreadleTesterPhase.scala
	modified:   src/main/scala/treadle/stage/phases/GetFirrtlAst.scala
	modified:   src/main/scala/treadle/stage/phases/HandleFormalStatements.scala
	modified:   src/main/scala/treadle/stage/phases/PrepareAst.scala
	modified:   src/main/scala/treadle/stage/phases/SetImplicitOutputInfo.scala
	modified:   src/main/scala/treadle/utils/AugmentPrintf.scala
	modified:   src/main/scala/treadle/utils/BitUtils.scala
	modified:   src/main/scala/treadle/utils/FindModule.scala
	modified:   src/main/scala/treadle/utils/FixupOps.scala
	modified:   src/main/scala/treadle/utils/VcdRunner.scala
	modified:   src/main/scala/treadle/vcd/VCD.scala
	modified:   src/main/scala/treadle/vcd/VCDConfig.scala
	modified:   src/main/scala/treadle/vcd/VcdOptions.scala
	modified:   src/main/scala/treadle/vcd/diff/VcdComparator.scala
	modified:   src/main/scala/treadle/vcd/diff/VcdDiffStage.scala
	modified:   src/test/scala/treadle/BitTwiddlingUtils.scala
	modified:   src/test/scala/treadle/CarryOrChain6.scala
	modified:   src/test/scala/treadle/ClockedManuallySpec.scala
	modified:   src/test/scala/treadle/CombinationalDelaySpec.scala
	modified:   src/test/scala/treadle/CoverageTest.scala
	modified:   src/test/scala/treadle/DynamicMemorySearch.scala
	modified:   src/test/scala/treadle/GCDTester.scala
	modified:   src/test/scala/treadle/InfoSpec.scala
	modified:   src/test/scala/treadle/LifeCellSpec.scala
	modified:   src/test/scala/treadle/MemoryInitializationTest.scala
	modified:   src/test/scala/treadle/MemoryUsageSpec.scala
	modified:   src/test/scala/treadle/PrintStopSpec.scala
	modified:   src/test/scala/treadle/RandomizeCircuitTest.scala
	modified:   src/test/scala/treadle/RegisterCycleTest.scala
	modified:   src/test/scala/treadle/RiscVMiniSimpleSpec.scala
	modified:   src/test/scala/treadle/ShiftRegisterSpec.scala
	modified:   src/test/scala/treadle/SimpleVendingMachineSpec.scala
	modified:   src/test/scala/treadle/SnapshotSpec.scala
	modified:   src/test/scala/treadle/StopBehaviorSpec.scala
	modified:   src/test/scala/treadle/TestUtils.scala
	modified:   src/test/scala/treadle/TestUtilsSpec.scala
	modified:   src/test/scala/treadle/VecSpec.scala
	modified:   src/test/scala/treadle/VerificationSpec.scala
	modified:   src/test/scala/treadle/asyncreset/AsyncResetBlackBox.scala
	modified:   src/test/scala/treadle/blackboxes/BlackBoxFinishSpec.scala
	modified:   src/test/scala/treadle/blackboxes/BlackBoxOutputSpec.scala
	modified:   src/test/scala/treadle/blackboxes/BlackBoxWithState.scala
	modified:   src/test/scala/treadle/chronometry/ClockCrossingSpec.scala
	modified:   src/test/scala/treadle/executable/ApplySugarProblemSpec.scala
	modified:   src/test/scala/treadle/executable/DataStoreSpec.scala
	modified:   src/test/scala/treadle/primops/AndOrXor.scala
	modified:   src/test/scala/treadle/primops/AndrOrrXorr.scala
	modified:   src/test/scala/treadle/primops/AsSIntAsUInt.scala
	modified:   src/test/scala/treadle/primops/CvtNegNot.scala
	modified:   src/test/scala/treadle/primops/EqOpsTester.scala
	modified:   src/test/scala/treadle/primops/ShlShrDshlDshr.scala
	modified:   src/test/scala/treadle/real/DspRealBlackBox.scala
	modified:   src/test/scala/treadle/vcd/VCDSpec.scala
	modified:   src/test/scala/treadle/vcd/VcdRoundTripTest.scala
```

Reply:
I just added the files that I changed and reset the rest and it seems to have done the job.

Reply:
Probably just `scalafmt`, I think that's run on just the 1 project (so the default project) while I think `scalafmtAll` is at least run on both main and test configurations of the project, and possibly, "run on all projects in this build".

Reply:
Oh sorry, I didn't scroll up and see that there are lots of `src/main` files edited too... In the past, it took like 2-3 runs of scalafmt to get future runs to be idempotent, perhaps we only ran it once here 🤷